Doc/library/sqlite3.rst

```diff
@@ -172,9 +172,13 @@ Module functions and constants
 
 .. function:: connect(database[, timeout, detect_types, isolation_level, check_same_thread, factory, cached_statements, uri])
 
-   Opens a connection to the SQLite database file *database*. You can use
-   ``":memory:"`` to open a database connection to a database that resides in RAM
-   instead of on disk.
+   Opens a connection to the SQLite database file *database* and return a
+   :class:`Connection` object.
```

Comment:
"return a Connection" is wrong, since it can be overridden by the *factory* parameter.

Either remove this addition, or rephrase it like: "Open a connection (...). By default, return a Connection object."